lib/opam_monorepo.ml

```diff
@@ -176,6 +178,15 @@ let initialize_switch ~network = function
       let open Obuilder_spec in
       [ run ~network "opam switch create %s" compiler_package ]
 
+let install_opam_provided_packages ~network ~cache ~lock_file_path ~lock_file_version =
+  let open Obuilder_spec in
+  match lock_file_version with
+  | V0_1 | V0_2 -> []
+  | V0_3 ->
+      [
+        run ~network ~cache "opam install --yes --deps-only ./%s" lock_file_path;
```

Comment:
The correct invocation should contain `--ignore-pin-depends`. Might be worth adding `--no-lint` to get rid of the output polution.

Reply:
Good point ­— added; my copy of opam 2.1.1 does not know `--no-lint` however so I left it out.